skyportal/facility_apis/observation_plan.py

```diff
@@ -357,17 +371,18 @@ def submit(request_id, asynchronous=True):
                     f"Created observation plan request for ID {plan.id} in session {plan._sa_instance_state.session_id}"
                 )
 
-                flow = Flow()
-                flow.push(
-                    '*',
-                    "skyportal/REFRESH_GCNEVENT_OBSERVATION_PLAN_REQUESTS",
-                    payload={"gcnEvent_dateobs": request.gcnevent.dateobs},
-                )
+                if request.gcnevent is not None:
+                    flow = Flow()
+                    flow.push(
+                        '*',
+                        "skyportal/REFRESH_GCNEVENT_OBSERVATION_PLAN_REQUESTS",
+                        payload={"gcnEvent_dateobs": request.gcnevent.dateobs},
+                    )
 
-                flow.push(
-                    '*',
-                    "skyportal/REFRESH_OBSERVATION_PLAN_NAMES",
-                )
+                    flow.push(
```

Comment:
We don't use that anymore I'm pretty sure. Right before submitting I think that we check if the queue name used is valid. But fetching the names of all existing plans was really not a good idea. Aside from being slow and mostly useless, it's a lot of data sent for mostly no good reason to everyone using the app

Reply:
You should be able to remove that

Reply:
but worth double checking the frontend component on submit method (for gcn obs plans)